docs/tutorials/go/audiobook/index.md

```diff
@@ -0,0 +1,1026 @@
+Spoken content helps people pass time, learn new things, and stay entertained without tying themselves to a screen or a book.
+Imagine turning your daily commute, workout sessions, or household chores into opportunities to catch up on _all_ your reading -- not just books.
+You can grab text from emails, from the web, or even that report you need to "read" for work.
+With the rise of advanced text-to-speech (TTS) technology, it's not just possible, it's quick to build in Go with simple APIs.
+
+For years, text-to-speech solutions have supported the assistive community by transforming words on the page into accessible engagement.
+However, until recently, robotic voices and unnatural rhythms made TTS unappealing for casual listeners.
+If you didn't need TTS, you probably wouldn't have used TTS.
+Now, companies like [OpenAI](https://openai.com/api/) have revolutionized speech synthesis.
+They created voices that sound like humans and not androids and released affordable APIs.
+Using these large language models (LLMs), you can enjoy warm, nuanced narration.
+So, wouldn't it be great to use this kind of tech to transform any text into an audiobook?
+
+In this tutorial, you'll use LLM voices to transform any text into audiobooks.
+You'll build a robust text-to-speech system with OpenAI APIs and Temporal Technology's error mitigation.
+Forget about manually checking statuses or handling errors.
+Just focus on converting text while Temporal makes sure everything runs smoothly.
+When complete, you'll be able to reliably convert any plain text file into an audiobook.
+
+Ready to transform your text into immersive audiobooks?
+Get started by checking that you have the necessary understanding, tools, and environment set up.
+
+## Prerequisites
+
+You can build the project by following this tutorial or, if you're feeling impatient, just grab the ready-to-go source from its [GitHub repo](https://github.com/temporalio/build-audiobook-go).
+This repository has the full source code and can serve as a guide or a fallback if you encounter issues when working through this tutorial.
+If you want to play first and explore later, you can come back and read through the how-to and background.
+Here's what you need to get going:
+
+1. **A local Go development environment**:
+   Follow ["Set up a local development environment"](/getting_started/go/dev_environment/) so you're ready to build Temporal applications with Go.
+   Ensure a local Temporal Service is running and that you can access the Temporal Web UI from port `8080`.
+   These services are necessary for you to build and run this project.
+
+1. **Basic understanding of the Temporal Go SDK**:
+   Work through the [Hello World in Go](/getting_started/Go/hello_world_in_Go/) tutorial.
+   This covers the basics of getting a Temporal Go SDK project up and running.
+
+1. **OpenAI API access**:
+   Sign up for your [OpenAI API developer account](https://openai.com/api/) and create a bearer token.
```

Comment:
Can we link directly to their docs for creating a token? It would be scope creep to walk through all of it here, but just linking to their homepage doesn't feel like enough.

Reply:
They're re-organizing from individual to organizational so I don't want to link to things that are changing. But very nice call. Thank you.